boranga/frontend/boranga/src/components/common/conservation_plans_flora_dashboard.vue

```diff
@@ -0,0 +1,1117 @@
+                    vm.column_effective_from_date,
+                    vm.column_effective_to_date,
+                    vm.column_next_review_date,
+                    // vm.column_family,
+                    // vm.column_genera,
+                    vm.column_status,
+                    vm.column_action,
+                ]
+                search = true
+            }
+
+            return {
+                autoWidth: false,
+                language: {
+                    processing: "<i class='fa fa-4x fa-spinner fa-spin'></i>"
+                },
+                order: [
+                    [0, 'desc']
+                ],
+                lengthMenu: [ [10, 25, 50, 100, -1], [10, 25, 50, 100, "All"] ],
+                responsive: true,
+                serverSide: true,
+                searching: search,
+                 //  to show the "workflow Status","Action" columns always in the last position
+                columnDefs: [
+                    { responsivePriority: 1, targets: 0 },
+                    { responsivePriority: 3, targets: -1 },
+                    { responsivePriority: 2, targets: -2 }
+                ],
+                ajax: {
+                    "url": this.url,
+                    "dataSrc": 'data',
+
+                    // adding extra GET params for Custom filtering
+                    "data": function ( d ) {
+                        d.filter_group_type = vm.group_type_name;
+                        d.filter_plan_type = vm.filterCPFloraType;
+                        d.filter_region = vm.filterCPFloraRegion;
+                        d.filter_district = vm.filterCPFloraDistrict;
+                        d.filter_application_status = vm.filterCPFloraApplicationStatus;
+                        d.filter_effective_from_date = vm.filterCPFloraEffectiveFromDate;
+                        d.filter_effective_to_date = vm.filterCPFloraEffectiveToDate;
+                        d.filter_review_date = vm.filterCPFloraReviewDate;
+                        d.is_internal = vm.is_internal;
+                    }
+                },
+                //dom: 'lBfrtip',
+                dom: "<'d-flex align-items-center'<'me-auto'l>fB>" +
+                     "<'row'<'col-sm-12'tr>>" +
+                     "<'d-flex align-items-center'<'me-auto'i>p>",
+                buttons: buttons,
+
+                columns: columns,
+                processing: true,
+                initComplete: function() {
+                    helpers.enablePopovers();
+                },
+            }
+        }
+    
+    },
+    methods:{
+        collapsible_component_mounted: function(){
+            this.$refs.collapsible_filters.show_warning_icon(this.filterApplied)
+        },
+        
+        fetchFilterLists: function(){
+            let vm = this;
+            //large FilterList of Species Values object
+            vm.$http.get(api_endpoints.filter_lists_species+ '?group_type_name=' + vm.group_type_name).then((response) => {
```

Comment:
please remove this api call if not using,

Reply:
Yaa, I think this will be needed once all the fields are confirmed for the conservation plans. If not, I'll remove then.